### ISSUE TITLE
This thing is dead in the water.

### ISSUE THREAD

**Issue by jimlynnjulian** [cinemagoer/cinemagoer#509]
Ran:
pytest tests/test_http_movie_combined.py

Result:
76 failed, 16 passed, 7 skipped in 58.65s

**Comment by jfadams1963** [comment]
Yeah. Some of my utilities work but getting filmographies is completely broken now and getting a list of similar titles is unreliable. At one time all my query tools worked fine.  

These are just broad-brushstroke observations. I haven't dug in to any trouble-shooting. Keeping Cinemagoer working nicely with iMDB seems to be a moving target.
